README.md

```diff
@@ -223,6 +223,8 @@ interface RuntimeOptions {
   module?: boolean;
   removeHTMLComments?: boolean;
   isMinified?: boolean;
+  initialize?: (sourceFile: SourceFile) => void;
+  finalize?: (sourceFile: SourceFile) => void;
```

Comment:
More than documenting these types, I could add a proper example somewhere.

Reply:
probably somewhere in /docs/api/AstAnalyser.md